test/format/config.sh

```diff
@@ -271,6 +271,8 @@ CONFIG configuration_list[] = {
 
 {"ops.verify", "configure verify", C_BOOL, 100, 1, 0}
 
+{"prefetch", "configure prefetch", C_BOOL, 50, 0, 0}
```

Comment:
What do people think of 50%?

Reply:
@etienneptl 50% seems reasonable to me given that we're not turning prefetch on by default when we configure it to be available on the connection level. Enabling pre-fetch 20% of the time for sessions also seems reasonable to me.

Reply:
I think 50% here is reasonable. My only comment is should this be under any other "format category"? Should it be at the very top-level like you did? Or could it be under `ops` (like `salvage` or 'verify')? Or maybe under `runs`? Or even under `wiredtiger`? There is definitely some conceptual overlap between those so I'm not sure exactly where, if anywhere, it best belongs. It sort of feels like `wiredtiger` might fit.

Reply:
I don't think it believes to `ops` as we don't call `session->prefetch`. However, it is kind of used the way `ops.bound_cursor` is used. We do pass an extra configuration when we want to use bounded cursors.

Regarding `runs`, it's seems to be very much for test/format feature, except `runs.in_memory`.

Finally, `wiredtiger` has three items:

1. config
2. rwlock
3. leak_memory

I don't see `prefetch` fitting here.

I like having `prefetch` the way it is as I could compare it to `chunk_cache` which has its own fields:
`chunk_cache`, `chunk_cache.capacity`, `chunk_cache.chunk_size`, etc.
I can prefetch going towards that direction too with a number of worker threads for example.

Let me know what you think.

Reply:
Ok. We can keep it at the top level.